src/util/pointer_expr.cpp

```diff
@@ -147,13 +147,13 @@ field_address_exprt::field_address_exprt(
   set(ID_component_name, component_name);
 }
 
-element_address_exprt::element_address_exprt(exprt base, exprt index)
+element_address_exprt::element_address_exprt(const exprt &base, exprt index)
   : binary_exprt(
-      std::move(base),
+      base, // used for type, can't move
```

Comment:
We have that situation in quite a few of the constructors, and don't usually include such a comment. So I'm wondering whether it might just unnecessarily alert people?